examples/simplesystem/source/events/button/clickedevent.d

```diff
@@ -0,0 +1,18 @@
+module events.button.clickedevent;
+
+import deventsystem.event : basicEventType, Event, EventType;
+
+@EventType("ClickedEvent")
+class ClickedEvent : Event
+{
+	mixin basicEventType!ClickedEvent;
+
+public:
+	this(in uint x, in uint y)
+	{
+		this.x = x;
+		this.y = y;
+	}
+
+	const uint x, y;
```

Comment:
Maybe you want `immutable` here. `unint` is a literal and you never modify the variable.

Reply:
Sure it's just I have nightmares with `immutable`. Also this is just an example, it's the user's choice how he wants to implement his events. `const` is the same as `immutable` in this case.

Reply:
> Sure it's just I have nightmares with immutable. 

I understand you. I also had some deep nightmares too. `const` is a soft thing, `immutable` is immutable.

Reply:
> const is the same as immutable in this case

Sure